web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -182,3 +182,28 @@ form .description
     &:before
       +icon-size(15px)
       content: '\e617'
+
+// Additional styling for dropbutton.
+.dropbutton-action
```

Comment:
Please keep this set of changes for ISAICP-3578. There are 2 tickets for separate cases, so please don't mix, it's hard to follow.